lib/linguist/languages.yml

```diff
@@ -3806,6 +3806,15 @@ Lua:
   interpreters:
   - lua
   language_id: 213
+Clue:
+  type: programming
+  tm_scope: source.clue
+  ace_mode: text
+  codemirror_mode: text
+  color: "#0009b5"
+  extensions:
+  - ".clue"
+  language_id: 163763508
```

Comment:
This entry needs to appear after [`Cloud Firestore Security Rules:`](https://github.com/github/linguist/blob/79c4aa4706e08f9eb36e46535f0e909f51f3b74d/lib/linguist/languages.yml#L1114-L1123), as `languages.yml` is alphabetised case-sensitively (in other words, ordered by code-point).